src/main/java/frc/bionic/swerve/AbstractDrivetrain.java

```diff
@@ -94,6 +97,17 @@ public void periodic() {
     swerveLF.periodic();
     swerveLR.periodic();
     swerveRR.periodic();
+
+    currentPose = odometry.update(Rotation2d.fromDegrees(getGyroAngle()),
+                                  swerveRF.getModuleState(), 
+                                  swerveLF.getModuleState(), 
+                                  swerveLR.getModuleState(), 
+                                  swerveRR.getModuleState());
+
+    SmartDashboard.putNumber("Current Pose X", currentPose.getTranslation().getX());
+    SmartDashboard.putNumber("Current Pose Y", currentPose.getTranslation().getY());
+    SmartDashboard.putNumber("Current Pose Rotation", currentPose.getRotation().getDegrees());
```

Comment:
If this was temporary stuff to put on the dashboard, it should probably be deleted now. If it's stuff that is long-term useful, it should be removed from here, and converted to Shuffleboard style in `initShuffleboard` and `syncShuffleboard` so that all of the meaningful dashboard stuff can be found in those two methods.

Reply:
That stuff should be long term, it's very useful for debugging, and eventually will be used for accuracy testing. I will try to fix that later today or tommorow.